adapters/syslog/syslog.go

```diff
@@ -55,6 +56,16 @@ func debug(v ...interface{}) {
 	}
 }
 
+func getHostname() string {
+	content, err := ioutil.ReadFile("/etc/host_hostname")
+	if err == nil && len(content) > 0 {
+		hostname = strings.TrimRight(string(content), "\r\n")
```

Comment:
This modifies the global hostname variable. Should we add a local declaration at the beginning of getHostname? (Or just return here instead of assigning the result?)